cleanup/tests/test_spawn.py

```
@@ -0,0 +1,136 @@
+            wait_seconds=3.0,
+            launcher_override=launcher,
+        )
+        assert ok, (
+            f"daemon did not come up; log: "
+            f"{log_path.read_text() if log_path.exists() else '<none>'}"
+        )
+        assert Path(sock_path).exists()
+
+        # Kill the fake daemon.
+        import subprocess
+        subprocess.run(["pkill", "-f", sock_path], check=False, timeout=2)
```

Ruff (0.15.12)

[error] `subprocess` call: check for execution of untrusted input

(S603)